src/util.cpp

```diff
@@ -488,6 +488,12 @@ namespace Sass {
 
       int cp = utf8::next(it, end);
 
+      // in case of \r, check if the next in sequence
+      // is \n and then advance the iterator.
+      if (cp == '\r' && it < end && utf8::peek_next(it, end) == '\n') {
```

Comment:
If CSS3 import only accepts network paths, then `\r` is legal string of length 2. :)
I have reverted it back to this original code. Now it aligns with Ruby both on Windows and OSX. CI seems to be happy too. :palm_tree: